notebooks/GoPro2CropKMZ.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>I had an error running this block of code. </p><p>The error reads</p><p> </p>

<pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/usr/local/lib/python3.10/dist-packages/pandas/core/indexes/base.py in get_loc(self, key)
   3652         try:
-&gt; 3653             return self._engine.get_loc(casted_key)
   3654         except KeyError as err:

</pre><p> </p><p> </p><p>7 frames</p>

<pre class="ql-syntax" spellcheck="false">pandas/_libs/hashtable_class_helper.pxi
 in pandas._libs.hashtable.Int64HashTable.get_item()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.Int64HashTable.get_item()

KeyError: 8
The above exception was the direct cause of the following exception:

KeyError                                  Traceback (most recent call last)
/usr/local/lib/python3.10/dist-packages/pandas/core/indexes/base.py in get_loc(self, key)
   3653             return self._engine.get_loc(casted_key)
   3654         except KeyError as err:
-&gt; 3655             raise KeyError(key) from err
   3656         except TypeError:
   3657             # If we have a listlike key, _check_indexing_error will raise

</pre><p>KeyError: 8</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>